ci.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import argparse
+import os
+import subprocess
+from contextlib import contextmanager
+from enum import Enum
+
+
+PANTS_INI = 'pants.ini'
+
+
+class PantsVersion(Enum):
+  unspecified = "unspecified"
+  pants_ini = "pants.ini"
```

Comment:
Slightly confusing to have an enum choice that looks like a filename. Maybe just `config`?